llvm/lib/SYCL/InSPIRation/InSPIRation.cpp

```diff
@@ -378,11 +369,6 @@ struct InSPIRation : public ModulePass {
 
           // Rename function name
           F.setName("sycl_func_" + Twine{funcCount++});
-
-          // Rename basic block name
-          // counter = 0;
-          // for (auto &B : F)
-          //   B.setName("label_" + Twine{counter++});
```

Comment:
I do not remember the context. So it is now useless to rename blocks?
By the way it would be interesting to test with `-g` and/or `-O0` because it might create some havoc in `xocc`...

Reply:
It seems xocc is fine without renamed block names at the moment, I have yet to require them at least, but I am always running -03 prior to passing it to xocc at the moment, so as you say it would be interesting to test that in the near future.

As for -g  it would be interesting however I'm not sure the runtime works with it yet (perhaps something to ask Intel). As I can compile an example for Intel with -g but when I run it I get a segfault. I suppose it doesn't prevent asking xocc to compile with -g though.